### DISCUSSION TITLE
GitHub Copilot could not connect to server. Extension activation failed: "User not authorized"

### DISCUSSION THREAD

**Discussion by biysaiy** [community/community#8294] [Copilot Conversations] [answered]
Sorry, I have problems. I applied for it last week, but I haven't received the email yet. Could you please help confirm whether IT is on the whitelist?

**Comment by biysaiy** [comment]
My guess is that it has something to do with github's Primary email address。because before Primary email address was incorrect

**Reply by jakobroenn02** [reply]
you simply don't have access yet. Wait for a couple of months and check again.

**Reply by biysaiy** [reply]
Oh... My friends‘s apply effective quiickly ... the next day

it's sampled ?

**Reply by jakobroenn02** [reply]
it might have something to do with your specs and the frequency in which you commit to a Github repo.

**Comment by davecheney** [comment, marked as answer]
@biysaiy i'm sorry you're still on the waitlist. We're adding users as we have capacity. Thank you for your patience and understanding.

**Reply by etemtolu** [reply]
I was on the Waiting List But 18 days passed and no response yet, Can you deal with the matter?

**Reply by etemtolu** [reply]
My Application Period Has Been More Than 1 Month

**Reply by CodeZ1LLa** [reply]
> My Application Period Has Been More Than 1 Month

My application period is 4 months, so...

**Reply by Onecoder13** [reply]
i am waiting for it

**Comment by mimo-x** [comment]
so we should use it after recived the official mail📧

**Comment by CodeZ1LLa** [comment]
Correct

**Comment by Rahman-Abdul** [comment]
my application is over two months now still on waiting list. please kindly help
